### ISSUE TITLE
Installation Not Processing Further

### ISSUE THREAD

**Issue by sabarivenkatesankrish** [DivanteLtd/open-loyalty#22]
Hi all,
I am trying to install the package through Docker. Its almost finished. It has installed npm, node, apache, php, postgres.. However, it still remains in the configuration of open_loyalty_mail as shown in the screenshot http://i.prntscr.com/Ha_LIN2FQCS_RQKzfHcDbw.png. It tooks almost 3 hours but still not get any further progress.
Any suggestion would be appreciated. 
Thanks,

**Comment by colejarczyk** [comment]
Hi,

This is expected behavior. It means everything went good and you can start using Open Loyalty.

**Comment by sabarivenkatesankrish** [comment]
Hi @colejarczyk,
Thanks for your comments.  When I am trying to proceed the next step(setup database, Elasticsearch and load some demo data) with "docker-compose exec backend phing setup" I am getting error like "ERROR: No container found for backend_1". Am I doing wrong? Kindly guide me how to proceed after this. Kindly have the screen shot for the error details(http://i.prntscr.com/BFgXODZQS6iNKEkxrLq-tg.png).
Thanks.

**Comment by sabarivenkatesankrish** [comment]
I tried the installation with the vagrant in my local server. The installation completed successfully. However the admin and customer panel is not working. I tried to access the same from the another system, since the installed one is not having GUI. I know something problem with defining the host name. I renamed the host name as "http://192.168.1.10" instead "http://127.0.0.1" in docket-compose.yml file. But this is not helped me. Kindly find the screenshot for further details.(http://i.prntscr.com/tQvM5-e-SFSEb58M18YDYQ.png). Anyone please suggest me to fix this.

**Comment by fareedpatel** [comment]
Hi @colejarczyk, I have installed the program through docker and once this is showing,

open_loyalty_mail | [APIv1] KEEPALIVE /api/v1/events

I am having the same problem as @sabarivenkatesankrish. ERROR: No container found for backend_1. Any ideas, thank you in advance.

**Comment by bizr** [comment]
Hello, I bumped onto the same issue but I don't know if it's the same cause here,
my problem was that the container was created but not started (I use portainer to manage, try it).
I read the detailed log docker-compose returned and I found my error, permission errors while running commands with sudo (that I should have run as my user), another day the same issue was up because my docker package installation was done by the Ubuntu repo which didn't work, official Docker repo release worked fine.

If you post us the detailed log of your docker-compose we can definately spot your erros guys but first try to see if your docker installation is the official and latest version imo.

Thanks.
